docs/specification/draft/basic/transports.md

```diff
@@ -48,38 +48,197 @@ sequenceDiagram
     deactivate Server Process
 ```
 
-## HTTP with SSE
+## Streamable HTTP
+
+{{< callout type="info" >}} This replaces the [HTTP+SSE
+transport]({{< ref "/specification/2024-11-05/basic/transports#http-with-sse" >}}) from
+protocol version 2024-11-05. See the [backwards compatibility](#backwards-compatibility)
+guide below. {{< /callout >}}
+
+In the **Streamable HTTP** transport, the server operates as an independent process that
+can handle multiple client connections. This transport uses standard HTTP with optional
+[Server-Sent Events](https://en.wikipedia.org/wiki/Server-sent_events) (SSE) for
+streaming multiple server messages. This permits "plain HTTP" MCP servers, as well as
+more feature-rich servers supporting streaming and server-to-client notifications and
+requests.
+
+The server **MUST** provide a single HTTP endpoint path (hereafter referred to as the
+**MCP endpoint**) that supports both POST and GET methods. For example, this could be a
+URL like `https://example.com/mcp`.
+
+### Message Exchange
+
+1. Every JSON-RPC message sent from the client **MUST** be a new HTTP POST request to the
+   MCP endpoint.
+
+2. When the client sends a JSON-RPC _request_ message to the MCP endpoint via POST:
```

Comment:
One interesting call-out here is to make sure that for cases where the client has a token for the server, it should also attach the `Authorization` header with a brarer token downstream, so that every invocation is able to access it (e.g., for scenarios where a tool can do an on-behalf-of token exchange).

Reply:
There should probably be a blanket statement in the auth RFC to say, "if the client has an auth token, it should include it in every MCP request".

Reply:
> There should probably be a blanket statement in the auth RFC to say, "if the client has an auth token, it should include it in every MCP request".

Yes, sorry, this was always the intention, we just hadn't made it clear enough.

Reply:
I'd suggest contemplating DPOP headers as well to prevent token replay

Reply:
My bad -- let me clarify -- 

"If the client has any HTTP authorization headers, it should include them in every MCP request"

Would future proof you for DPOP headers

Reply:
Ah, got it, thank you!